README.md

```diff
@@ -1,46 +1,79 @@
+## 👏 Contribute to WinUI Gallery
 
-To obtain information about Windows 11 development, go to the [Windows Dev Center](https://developer.microsoft.com/windows)
+Any samples or docs improvements you'd like to see? We're always looking for a helping hand. Feel free to file an issue to start the discussion, or even better, create a PR with the change you'd like to see!
 
-To obtain information about Microsoft Visual Studio and the tools for developing Windows apps, go to [Visual Studio](http://go.microsoft.com/fwlink/?LinkID=532422)
+Check out these [help wanted](https://github.com/microsoft/WinUI-Gallery/issues?q=is%3Aopen+is%3Aissue+label%3A%22help+wanted%22) issues if you don't know where to start.
 
-To contact the authors, please reach out to ControlsGallery@microsoft.com
+You can also learn more about current happenings with WinUI Gallery in the [project board](https://github.com/orgs/microsoft/projects/368).
 
+## 🚀 Getting Started
+Quick start guide to building WinUI 3 Gallery locally:
 
+### 1. Set up the Environment
 
-## Related topics
+>**Note**: The WinUI 3 Gallery requires [Visual Studio 2022](https://visualstudio.microsoft.com/vs/) or later to build and Windows 10 or later to execute.
+If you're building an app with the Windows App SDK for the first time, follow the [installation instructions here.](https://learn.microsoft.com/en-us/windows/apps/windows-app-sdk/set-up-your-development-environment)
 
-[Get started with Windows 11 apps](https://docs.microsoft.com/windows/apps/get-started)  
+**Required [Visual Studio components](https://learn.microsoft.com/en-us/windows/apps/windows-app-sdk/set-up-your-development-environment?tabs=cs-vs-community%2Ccpp-vs-community%2Cvs-2022-17-1-a%2Cvs-2022-17-1-b#required-workloads-and-components):**
+- .NET Desktop Development.
+- Windows App SDK C# Templates
+- Windows 10 SDK (10.0.19041.0)
 
-[Install a prebuilt version of this app from Microsoft Store](https://www.microsoft.com/store/productId/9P3JFPWWDZRC). Each control page in the application has links to relevant Microsoft Docs for that control.
+### 2. Clone the Repository
 
-[Windows UI Library (WinUI)](https://docs.microsoft.com/windows/apps/winui)
+```shell
+git clone https://github.com/microsoft/WinUI-Gallery.git
+```
 
+### 3. Open .sln with Visual Studio and Build!
+```shell
+.\WinUIGallery\WinUIGallery.DesktopWap.sln
+```
+Make sure to select `WinUIGallery.DesktopWap.Package` for startup project in Visual Studio.
+
+## ➡️ Further information
+
+To obtain information about Windows 11 development, go to the [Windows Dev Center](https://developer.microsoft.com/windows)  
+To obtain information about Microsoft Visual Studio and the tools for developing Windows apps, go to [Visual Studio](http://go.microsoft.com/fwlink/?LinkID=532422)  
+To contact the authors, please reach out to ControlsGallery@microsoft.com
+
+### Related topics
+
+[Get started with Windows 11 apps](https://docs.microsoft.com/windows/apps/get-started)  

```

Comment:
```suggestion
[Get started with building WinUI apps](https://learn.microsoft.com/en-us/windows/apps/get-started/start-here)  

```

Reply:
I don't think we want en-us be part of the URL since it breaks automatic locale logic on the learn/docs pages (see #151 )

Reply:
Thanks, good catch!